src/main/java/org/sonar/plugins/groovy/foundation/Groovy.java

```diff
@@ -20,18 +20,52 @@
 
 package org.sonar.plugins.groovy.foundation;
 
+import com.google.common.collect.Lists;
+import org.apache.commons.lang.StringUtils;
+import org.sonar.api.config.Settings;
 import org.sonar.api.resources.AbstractLanguage;
+import org.sonar.plugins.groovy.GroovyPlugin;
+
+import java.util.List;
 
 public class Groovy extends AbstractLanguage {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2160)